charts/registry-scanner/templates/cronjob.yaml

```diff
@@ -25,7 +25,6 @@ spec:
           labels:
             app.kubernetes.io/name: {{ include "registry-scanner.name" . }}
             app.kubernetes.io/instance: {{ .Release.Name }}
-            {{- include "registry-scanner.selectorLabels" . | nindent 12 }}
```

Comment:
I see the `selectorLabels`already defines `app.kubernetes.io/name` and `app.kubernetes.io/instance` labels, and seems to be the standard way in _helpers.tpl and other Helm charts to define the standard set of labels. It is used in the deployment template, and then in the service to match or "select" the corresponding pods by labels.

What I would suggest is keeping the include (so the line you just removed) and instead remove lines 26 and 27, as those labels are already provided in the `registry-scanner.selectorLabels` template, what do you think?

Reply:
Thanks for suggestion,
I've found `registry-scanner.labels` that includes `registry-scanner.selectorLabels` this should be the way.